### PR TITLE
Updates translation to spanish of bootstrap

### LINKED ISSUE #6498: Translation to Spanish: Verify and update translation of bootstrap.json
- [ ]  Use Responsive Design with Bootstrap Fluid Containers
- [ ]  Make Images Mobile Responsive
- [ ]  Center Text with Bootstrap
- [ ]  Create a Bootstrap Button
- [ ]  Create a Block Element Bootstrap Button
- [ ]  Taste the Bootstrap Button Color Rainbow
- [ ]  Call out Optional Actions with Button Info
- [ ]  Warn your Users of a Dangerous Action
- [ ]  Use the Bootstrap Grid to Put Elements Side By Side
- [ ]  Ditch Custom CSS for Bootstrap
- [ ]  Use Spans for Inline Elements
- [ ]  Create a Custom Heading
- [ ]  Add Font Awesome Icons to our Buttons
- [ ]  Add Font Awesome Icons to all of our Buttons
- [ ]  Responsively Style Radio Buttons
- [ ]  Responsively Style Checkboxes
- [ ]  Style Text Inputs as Form Controls
- [ ]  Line up Form Elements Responsively with Bootstrap
- [ ]  Create a Bootstrap Headline
- [ ]  House our page within a Bootstrap Container Fluid Div
- [ ]  Create a Bootstrap Row
- [ ]  Split your Bootstrap Row
- [ ]  Create Bootstrap Wells
- [ ]  Add Elements within your Bootstrap Wells
- [ ]  Apply the Default Bootstrap Button Style
- [ ]  Create a Class to Target with jQuery Selectors
- [ ]  Add ID Attributes to Bootstrap Elements
- [ ]  Label Bootstrap Wells
- [ ]  Give Each Element a Unique ID
- [ ]  Label Bootstrap Buttons
- [ ]  Use Comments to Clarify Code

### PR DESCRIPTION
**Pull request by vtamara**
Closes #6498

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/bootstrap.json [hunk 1]
```diff
@@ -1025,7 +1025,7 @@
       "challengeType": 0,
       "nameEs": "Crea un encabezado personalizado",
       "descriptionEs": [
-        "Crearemos un encabezado simple para nuestra aplicación de fotos de gatos poniéndolos en la misma línea.",
+        "Crearemos un encabezado simple para nuestra aplicación de fotos de gatos poniéndo el título y la relajante imagen del gato en la misma línea.",
```
Comment (justinian336) [inline seed/challenges/01-front-end-development-certification/bootstrap.json]: `poniendo` en lugar de `poniéndo`

File: seed/challenges/01-front-end-development-certification/bootstrap.json [hunk 2]
```diff
@@ -1959,7 +1964,8 @@
       "nameEs": "Dale a cada elemento un ID único",
       "descriptionEs": [
         "También quisiéramos referirnos a cada botón por su id utilizando jQuery.",
-        "Dale a cada uno de tus botones un id único, comenzando con <code>target1</code> y terminando con <code>target6</code>."
+        "Dale a cada uno de tus botones un id único, comenzando con <code>target1</code> y terminando con <code>target6</code>.",
+        "Asegurate que <code>target1</code> a <code>target3</code> queden en <code>#left-well</code>, mientras que <code>target4</code> a <code>target6</code> queden en <code>#right-well</code>."
```
Comment (justinian336) [inline seed/challenges/01-front-end-development-certification/bootstrap.json]: `asegúrate` en vez de `asegurate`

**Comment by justinian336** [comment]
Just two small corrections. Almost there @vtamara!

**Comment by vtamara** [comment]
Thanks @justinian336.  This one is ready to be merged.

**Comment by bugron** [comment]
@vtamara @justinian336 thanks, merged!

**Pull request merged**